aten/src/ATen/native/CPUBlas.cpp

```diff
@@ -337,9 +338,20 @@ void gemm(
               b, &ldb_,
               &beta_,
               float_v.data(), &ldc_);
-      for (auto cv: float_v) {
-        *(c++) = c10::convert<at::BFloat16>(cv);
-      }
+      at::parallel_for(0, c_size, 1, [&](int64_t begin, int64_t end) {
+        int64_t i = begin;
+        //Vectorized Loop
+        for (; i + c_size <= end; i += c_size) {
+            auto a_vec = at::vec::Vectorized<float>::loadu(&float_v[i]); // Load vec_size floats
```

Comment:
Using `Vectorized` outside of the `ATen/native/cpu/` directory will only use SSE. You would need to have a cpu kernel behind a `DispatchStub` to get AVX2 or AVX512 support.